spec/cli/yardoc_spec.rb

```diff
@@ -510,10 +510,23 @@ def foo; end
       optsdata = String.new("foo bar")
       expect(optsdata).to receive(:shell_split)
       expect(File).to receive(:read_binary).with("test").and_return(optsdata)
+      allow(optsdata).to receive(:gsub).and_return(optsdata)
```

Comment:
The `optsdata` string is frozen in this any many other tests.  Since shell split is an extension on `String`  I can't mock there.